ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -132,6 +143,9 @@
   :trust-password       - the password to the truststore
   :max-threads          - the maximum number of threads to use (default 50)
   :min-threads          - the minimum number of threads to use (default 8)
+  :max-queued-requests  - the maximum number of requests to be queued (default INTEGER/MAX_VALUE)
```

Comment:
I think we can remove the default here from the docs. `Integer/MAX_VALUE` is the same as not having a maximum, for all practical purposes.